# Add flag --hide-crashed-bubble to hide the bubble box:
# "Restore Pages? Chromium didn't shut down correctly."

--- a/chrome/browser/ui/startup/startup_browser_creator_impl.cc
+++ b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
@@ -551,7 +551,8 @@ void StartupBrowserCreatorImpl::AddInfoB
   if (command_line_.HasSwitch(switches::kKioskMode))
     return;
 
-  if (HasPendingUncleanExit(browser->profile()))
+  if (HasPendingUncleanExit(browser->profile()) &&
+      !command_line_.HasSwitch("hide-crashed-bubble"))
     SessionCrashedBubble::ShowIfNotOffTheRecordProfile(browser);
 
   // The below info bars are only added to the first profile which is launched.
--- a/chrome/browser/ungoogled_flag_entries.h
+++ b/chrome/browser/ungoogled_flag_entries.h
@@ -32,4 +32,8 @@
      "Show avatar/people/profile button",
      "Show avatar/people/profile button in the browser toolbar.  ungoogled-chromium flag.",
      kOsDesktop, MULTI_VALUE_TYPE(kShowAvatarButtonChoices), kBreezeFlag},
+    {"hide-crashed-bubble",
+     "Hide crashed bubble",
+     "Hides the bubble box with the message \"Restore Pages? Chromium didn't shut down correctly.\" that shows on startup after the browser did not exit cleanly.  ungoogled-chromium flag.",
+     kOsAll, SINGLE_VALUE_TYPE("hide-crashed-bubble"), kBreezeFlag},
 #endif  // CHROME_BROWSER_UNGOOGLED_FLAG_ENTRIES_H_
